src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -16,6 +16,7 @@
     <DefineConstants>$(DefineConstants);ASPNETODATA;ASPNETWEBAPI;NETFX;NETFX45</DefineConstants>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Microsoft.CSharp" />
```

Comment:
>"Microsoft.CSharp" /> [](start = 23, length = 21)

why add this?

Reply:
The way I found out to deal with generic of variable type argument is using dynamic, and VS requires Microsoft.CSharp.
[https://stackoverflow.com/questions/11417047/c-sharp-dynamic-compilation-and-microsoft-csharp-dll-error](https://stackoverflow.com/questions/11417047/c-sharp-dynamic-compilation-and-microsoft-csharp-dll-error)

---
In reply to: [190750826](https://github.com/OData/WebApi/pull/1445#discussion_r190750826) [](ancestors = 190750826)